clerk/authorization_test.go

```diff
@@ -0,0 +1,90 @@
+package clerk
+
+import (
+	"testing"
+
+	"github.com/go-jose/go-jose/v3"
+)
+
+func TestClient_CheckAuthorization(t *testing.T) {
+	c, _ := NewClient("token")
+	token, pubKey := testGenerateTokenJWT(t, dummySessionClaims, "kid")
+	client := c.(*client)
+	client.jwksCache.set(testBuildJWKS(t, pubKey, jose.RS256, "kid"))
+
+	// user has permission
+	hasPermission, err := c.CheckAuthorization(
+		token,
+		CheckAuthorizationParams{Permission: "org:billing:manage"},
+	)
+	if err != nil {
```

Comment:
🔧 We can use the `testify` package in tests:

```suggestion
	require.NoError(t, err)
```